functions/go/inflate-helm-chart/third_party/sigs.k8s.io/kustomize/api/builtins/HelmChartInflationGenerator.go

```diff
@@ -0,0 +1,336 @@
+// Copyright 2021 The Kubernetes Authors.
```

Comment:
Since we fork the code, I guess there are some modification, right?
When we fork code, we usually do it in a separate PR. If possible, it should simply copy the code w/o any modification.
Any modification should be done in a separate PR.